app/views/exam_templates/assign_errors.html.erb

```diff
@@ -0,0 +1,279 @@
+<% content_for :head do %>
+    <%= stylesheet_link_tag('grader') %>
+    <%= stylesheet_link_tag('pdfjs/viewer') %>
+    <%= stylesheet_link_tag('context_menu') %>
+
+    <%= javascript_include_tag 'Results/file_selector',
+                               'panes',
+                               'pdfjs',
+                               'testresults_config',
+                               'Results/context_menu' %>
+
+    <%= javascript_tag do -%>
+        PDFJS.workerSrc = "<%= @host %>/assets/pdfjs_lib/pdf.worker.js";
+    <% end -%>
+<% end %>
+
+<% # SourceCodeGlower, ImageAnnotations, and SyntaxHighlighter brushes %>
```

Comment:
Yikes, did I miss in on Assign scans? You shouldn't need the syntax highlighter stuff in either view. (Remove it from this view in this branch, make a separate branch for the other view.)